modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_entries_admin/js/AccountEntriesManagementToolbarPropsTransformer.js

```diff
@@ -20,7 +20,7 @@ const updateAccountEntries = (portletNamespace, url) => {
 	if (form) {
 		postForm(form, {
 			data: {
-				accountEntryIds: Liferay.Util.listCheckedExcept(
+				accountEntryIds: Liferay.Util.getCheckedCheckboxes(
```

Comment:
For these instances where we are calling it within a JS file, we should be calling it via an import instead of referencing the global util.

import {getCheckedCheckboxes} from 'frontend-js-web'

Reply:
Yea, can do!